Add a reference on returning fields during a search. (#57500)
elastic/elasticsearch: docs/reference/search/search-fields.asciidoc

[discrete]
[[search-fields]]
=== Return fields in a search

By default, each hit in the search response includes the document

> <<mapping-source-field,`_source`>>, which is the entire JSON object that was

Reviewer: Nit: Just breaking this sentence up for easier reading. Fine as-is though.
```suggestion
<<mapping-source-field,`_source`>>. The `source` contains the entire JSON object 
```

> provided when indexing the document. Often only certain fields are required in
> the response. In this case you can use
> <<request-body-search-source-filtering,source filtering>> to restrict what
> parts of the source are returned.

Reviewer: Reword for easier reading. If you keep this as-is, I would add a comma to `In this case,`.
```suggestion
If you only need certain fields in the search API's response, you can use
<<request-body-search-source-filtering,source filtering>> to restrict what
parts of the source are returned.
```

> Returning fields using the document source has some limitations:

Reviewer: ```suggestion
Returning fields using only the document source has some limitations:
```

* The `_source` field does not include <<multi-fields, multi-fields>> or
<<alias, field aliases>>. Likewise, a field in the source will not contain
values that were copied into it through `copy_to`.
* Since the `_source` is stored as a single field in Lucene, the whole source

> object must be loaded and parsed, even if only a small number of fields is needed.

Author: That's good to know!

> Elasticsearch supports some alternative methods for returning fields that help

Reviewer: ```suggestion
{es} supports some alternative methods for returning fields that can help you
```
